docs/content/reference/resource-schema/link-schema/dapr-schema/dapr-pubsub/snippets/dapr-pubsub-values.bicep

```
@@ -34,7 +35,7 @@ resource pubsub 'Applications.Link/daprPubSubBrokers@2022-03-15-privatepreview'
   properties: {
     environment: environment
     application: app.id
-    mode: 'values'
+    resourceProvisioning: 'manual'    
```

nit: remove extra spaces.
```suggestion
    resourceProvisioning: 'manual'
```